runtime/storage/firebase-storage.js

```diff
@@ -10,6 +10,7 @@ import {StorageProviderBase} from './storage-provider-base.js';
 import {firebase} from '../../platform/firebase-web.js';
 import {assert} from '../../platform/assert-web.js';
 import {KeyBase} from './key-base.js';
+import {atob} from '../../platform/atob-web.js';
```

Comment:
Add a test that would have caught the absence of this?

Reply:
There is one, now that the decodeKey function is actually used.